### PR TITLE
Add long permalink to share dialog.

### PR DESCRIPTION
**Pull request by caseypt**
* Allow a user to access the unshortened version of the permalink
for the purposes of configuring the launchpad.

Refs #291

### CODE REVIEW

**Comment by caseypt** [comment]
I'm not sure if there is more to this than what I did. This addresses the description of #296, but not necessarily the title, which references "admin users".

**Comment by caseypt** [comment]
selection_032

selection_033

File: src/GeositeFramework/js/Permalink.js [hunk 1]
```diff
@@ -108,6 +109,16 @@
 
         updateSize: function (key) {
             this.model.set(key, this.$('.' + key).val());
+        },
+
+        toggleLongPermalink: function() {
+            var $textbox = this.$el.find('#long-permalink-textbox');
+
+            if ($textbox.is(':hidden')) {
```
Comment (steventlamb) [inline src/GeositeFramework/js/Permalink.js]: does `$textbox.toggle()` work here?

**Comment by steventlamb** [comment]
LGTM. I don't have an environment set up to exercise this branch at the moment though.

File: src/GeositeFramework/js/Permalink.js (hunk 1 above)
Comment (caseypt) [inline src/GeositeFramework/js/Permalink.js]: Right. Forgot about that. Thanks

**Comment by caseypt** [comment]
The launchpad config [only needs](https://github.com/CoastalResilienceNetwork/GeositeFramework/commit/9b4a705bf82727c424a23b43cbdd949c0f8f22f1#diff-3a581756cbee3b507ef56092b7661596R102) the hash from the URL. Any opinions on whether the new input should include the URL prefix + the hash, or just the hash.

**Comment by kdeloach** [comment]
I think it's fine as-is.

**Comment by kdeloach** [comment]
+1

**Pull request merged**